.github/workflows/spelling-all.yaml

```diff
@@ -0,0 +1,30 @@
+name: Check Spelling - All
+
+on:
+  push:
+    branches:
+      - main
+      - RPSaaSMaster
+      - typespec-next
```

Comment:
```suggestion
```

I don't think it's worthwhile to run this on `typespec-next`, the PR validation should be sufficient.